code/modules/mob/living/carbon/human/human.dm

```diff
@@ -408,6 +408,7 @@
 							DT.dogtag_taken = TRUE
 							DT.icon_state = "dogtag_taken"
 							var/obj/item/dogtag/D = new(loc)
+							D.fallen_references = list(DT.registered_ref?.resolve())
```

Comment:
since tgui strip menu isn't this deprecated and handled in `/datum/strippable_item/mob_item_slot/id/alternate_action()`?

Reply:
it sure as hell is. do you want me to take this out or leave it in for whatever reason?

Reply:
i'm mostly confused as to why this whole topic proc is here but that's not your fault. i guess leave it for parity but SOMEONE really should delete all this